src/common/resolve.js

```diff
@@ -1,12 +1,18 @@
 "use strict";
 
-let { resolve } = eval("require");
+// eslint-disable-next-line prefer-destructuring
+let resolve = eval("require").resolve;
```

Comment:
We should add comment explain why this is disabled, because this can't understand `/scripts/build/babel-plugins/transform-custom-require.js` 

I'm not near my laptop, can you do this?

Reply:
ok